src/sysroot.rs

```diff
@@ -276,16 +276,12 @@ pub fn update(
         &dst,
     )?;
 
-    let bin_dst = lock.parent().join("bin");
-    util::mkdir(&bin_dst)?;
-    util::cp_r(
-        &sysroot
-            .path()
-            .join("lib/rustlib")
-            .join(&meta.host)
-            .join("bin"),
-        &bin_dst,
-    )?;
+    let bin_src = sysroot.path().join("lib/rustlib").join(&meta.host).join("bin");
+    if bin_src.exists() {
```

Comment:
Could you add a comment explaining why this is necessary, e.g., mention a `$TRIPLE` that has no `bin`? I am a bit confused why we need to copy this at all if `bin` is seemingly an optional directory.

Reply:
I don't really remember. I just ran into this error once, noticed that I could create an empty directory that would make everything work, and filed this PR.

Reply:
Looking on my filesystem, looks like this is where `lld` or `rust-lld` lives.